src/Riok.Mapperly/Emit/SyntaxFactoryHelper.cs

```diff
@@ -350,4 +353,32 @@ private static IEnumerable<SyntaxNodeOrToken> Join(SyntaxToken sep, bool insertT
             yield return sep;
         }
     }
+
+    public static IdentifierNameSyntax FullyQualifiedIdentifier(ITypeSymbol typeSymbol)
+    {
+        if (typeSymbol is null)
+            throw new ArgumentNullException(nameof(typeSymbol));
```

Comment:
This code base uses nullable reference types and therefore doesn't check if an argument is null. This check should also be removed to match the style of this codebase.

Reply:
give me 2 mint. I'll update it

Reply:
updated